utils/runtime.go

```diff
@@ -0,0 +1,110 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+	"github.com/go-redis/redis/v7"
+	sharedpb "github.com/tron-us/go-btfs-common/protos/shared"
+	"github.com/tron-us/go-common/db/postgres"
+	dbenv "github.com/tron-us/go-common/env/db"
+	"github.com/tron-us/go-common/log"
+	"go.uber.org/zap"
+	"os"
+	"strconv"
+)
+
+const (
+	DBConnectionHealthy = "DB Connection Healthy"
+	DBWriteConnectionError = "Cannot connect to the write database!"
+	DBReadConnectionError  = "Cannot connect to the read database!"
+	DBEnvironmentError = "Cannot parse the database environment variable"
+	RedisEnvironmentError = "Cannot parse the redis database environment variable"
+	RedisConnectionError = "Cannot connect to the redis database!"
+)
```

Comment:
use go-common constant package instead, plz see status-server implementation for example

Reply:
ok